src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java

```diff
@@ -0,0 +1,127 @@
+package com.imagination.cbs.service.helper;
+
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import com.imagination.cbs.domain.ApprovalStatusDm;
+import com.imagination.cbs.domain.Booking;
+import com.imagination.cbs.domain.BookingRevision;
+import com.imagination.cbs.exception.CBSApplicationException;
+import com.imagination.cbs.exception.CBSUnAuthorizedException;
+import com.imagination.cbs.security.CBSUser;
+import com.imagination.cbs.service.AdobeSignService;
+import com.imagination.cbs.service.Html2PdfService;
+import com.imagination.cbs.service.LoggedInUserService;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BookingHrApproveHelperTest {
+
+	@InjectMocks
+	private BookingHrApproveHelper bookingHrApproveHelper;
+	
+	@Mock
+	private LoggedInUserService loggedInUserService;
+
+	@Mock
+	private Html2PdfService html2PdfService;
+
+	@Mock
+	private AdobeSignService adobeSignService;
+
+	@Mock
+	private BookingSaveHelper bookingSaveHelper;
+
+	@Mock
+	private EmailHelper emailHelper;
+	
+	@Test
+	public void shouldPrepareMailAndSendToHrWithBookingRevisionDetailsWhenUserIsAuthorized() throws IOException {
```

Comment:
can we break this test in multiple small test cases, having big test cases those test multiple things at one place is not recommended.

Reply:
Only one condition is included in this test that Authorized user can send email to HR.. and for UnAuthorized there is another test case, so not possible to break this test